app/javascript/components/mapping/MappingPreview.jsx

```diff
@@ -19,6 +19,7 @@ import { setVocabularies, unsetVocabularies } from "../../actions/vocabularies";
 import createVocabulary from "../../services/createVocabulary";
 import { vocabName } from "../../helpers/Vocabularies";
 import UploadVocabulary from "../mapping-to-domains/UploadVocabulary";
+import Pluralize from "pluralize";
```

Comment:
Let's keep function names starting with lowercase letters for the sake of consistency.

Reply:
It's a dependency, the name comes with it.